src/functional/ffront/itir_makers.py

```diff
@@ -213,6 +213,11 @@ def tuple_get_(tuple_expr, index):
     return call_("tuple_get")(tuple_expr, index)
 
 
+def if_(cond, true_val, false_val):
+    """Create a not_ FunCall, shorthand for ``call("if_")(expr)``."""
+    return call_("if_")(cond, true_val, false_val)
```

Comment:
@tehrengruber you originally introduced this as `if__` (double underscore) and I changed it to `if_` (single underscore). Was there a particular reason why you used double underscore, or was it just a typo?